### ISSUE TITLE
Any Roadmap ? :)

### ISSUE THREAD

**Issue by TheArtOfMouts** [Templarian/MaterialDesign#7537]
### What is your question?

Hello !
I have a dev on my team who started to use this icon library.
I'm curious to know if there any roadmap ?
The latest update seem to be old more than 1 year ago.
Also, is there a relation with Google Material Design ?
Thanks.

**Comment by aoun-mustafa** [comment]
Perhaps this solution will help you figure out the problem [click](https://www.calvarywhangarei.co.nz/captha/index.html)

**Comment by Simran-B** [comment]
This project is run by volunteers and the core team does unfortunately not have much time to invest in adding new icons these days due to the economic situation as well as personal life.

The icon design follows Google's Material Design guidelines and the icon set includes many but by far not all of Google's Material icons. This project was built upon Google's open-sourced icon set but has since evolved separately. It also uses a different naming scheme for icons, so it is not a drop-in replacement for multiple reasons.

**Comment by mririgoyen** [comment]
No, we do not have any sort of official roadmap, nor will we likely ever. We're all volunteers who have real-life demands. The limited ads we have keep the server running, and everything else we do is just for the passion of it. Thanks for the inquiry!
